src/pages/AddLiquidity/index.tsx

```diff
@@ -137,11 +138,12 @@ export default function AddLiquidity({
     baseCurrency ?? undefined,
     existingPosition
   )
-  const usdcBaseCurrencyPriceInNumber = Number(baseCurrencyPriceInUSDC?.toSignificant(4))
-  const quoteCurrencyPrice =
-    usdcBaseCurrencyPriceInNumber * Number(invertPrice ? price?.invert().toSignificant(4) : price?.toSignificant(4))
-  const roundedQuoteCurrencyPrice = isNaN(usdcBaseCurrencyPriceInNumber) ? undefined : quoteCurrencyPrice.toFixed(2)
-  const showBaseCurrencyPrice = isEthStablecoinPair(baseCurrency ?? undefined, quoteCurrency ?? undefined)
+
+  const quoteCurrencyAmount = invertPrice ? price?.invert().toSignificant(6) : price?.toSignificant(6)
```

Comment:
would love another set of eyes on this math here 👀  i believe this is accomplishing the same thing as before, but simpler?